site/content/docs/main/volume-group-snapshots.md

```diff
@@ -0,0 +1,266 @@
+---
```

Comment:
Let's also clarify the coordination between VGS and data mover. E.g., how do users use VGS+data mover?